package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-native-webview",
-  "description": "React Native WebView component for iOS, Android, and Windows 10 (coming soon)",
+  "description": "React Native WebView component for iOS, Android, macOS, and Windows 10 (coming soon)",
```

Comment:
We should remove the Windows 10 reference, as we no longer plan to support it (unless someone steps up to champion it.)